src/helpers/snapshot.ts

```diff
@@ -4,20 +4,27 @@ import { formatProposals, switchStrategiesAt } from '@/helpers/utils';
 import { getProfiles } from '@/helpers/profile';
 import getProvider from '@snapshot-labs/snapshot.js/src/utils/provider';
 import client from '@/helpers/client';
-import { getProposalData } from '@/helpers/graphql';
+import { apolloClient } from '@/apollo';
+import { PROPOSAL_AND_VOTES_QUERY } from '@/helpers/queries';
+import { cloneDeep } from 'lodash';
 
 export async function getProposal(space, id) {
   try {
     console.time('getProposal.data');
     const provider = getProvider(space.network);
     const response = await Promise.all([
-      getProposalData(id),
+      apolloClient.query({
+        query: PROPOSAL_AND_VOTES_QUERY,
+        variables: {
+          id
+        },
+        fetchPolicy: 'network-only'
```

Comment:
Shouldn't this be `no-cache`? With `network-only` it will still store a cache, but i don't see when the cache is being used. We could  break this request in 2 actually, proposal (with cache) and votes (without cache).

Reply:
Sounds good